dvc/commands/add.py

```diff
@@ -119,6 +120,13 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Override local file or folder if exists.",
     )
+    parser.add_argument(
+        "--no-relink",
+        dest="relink",
+        action="store_false",
+        help="Don't recreate links from cache to workspace.",
+    )
```

Comment:
I'd avoid introducing this asymmetry here. 

(What you have done is useful if you have support for both negative and positive behavors. Eg: you'd see this in `git`: `git stash --include-untracked` and `git stash --no-include-untracked`).

```suggestion
    parser.add_argument(
        "--no-relink",
        default=False,
        action="store_true",
        help="Don't recreate links from cache to workspace.",
    )
```

Reply:
Same in `commit`.

Reply:
@skshetry Should we add both? We already have `dvc checkout --relink`. We could have `--relink` and `--no-relink` options for all of those commands even if it's redundant when it's the default.

Reply:
We haven't done anything like that before in dvc. So maybe better to not introduce them. But up to you.

Reply:
I'm seeing plenty of examples of either `store_false` and `store_true` in the codebase for similar situations, and I think it is just as awkward to use `store_true` (I will have to do `relink=not self.args.no_relink`), so I'm going to leave this as is.